fileuploader.config.example.toml

```diff
@@ -1,9 +1,16 @@
 [Server]
 ListenAddress = "127.0.0.1:8088"
+# Note: ListenAddress is not used when running as a webircgateway plugin.  In
+# those situations, the listen addresses of the webircgateway will be used,
+# including HTTPS if configured.
+
 BasePath = "/files"
 # BasePath = "https://example.com/files" # external URL for use behind reverse proxy
 # CorsOrigins = [ "http://example.com" , "https://example.org" ]
 
+# When running as a webircgateway plugin, this path will be relative to the
+# webircgateway domain, e.g. https://ws.irc.example.com/files
+
```

Comment:
Based on the instructions in `README.md`, it seems like `CorsOrigins` may also need to be set, e.g.

```toml
CorsOrigins = [ "https://irc.example.com" ]
```

(And **not** `ws.irc.example.com`)

Is this correct?  If so, I should update this.

Reply:
That is correct. I've just added an explanation for CORS to the example config now, thanks for pointing that out.